expense-splitter/src/components/widgets/GetStarted.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useRef } from "react";
+import { UseDataContext } from "../context/SiteContext";
+import Button from "../ui/Button";
+import Dialog from "../ui/Dialog";
+
+const GetStarted = () => {
+  const { friends, handleSetModal } = UseDataContext();
+
+  useEffect(() => {
+    if (friends.length < 2) {
+      console.log("Get started by adding a friend to split expenses with.");
+      toggleDialog(deleteDialogRef);
+    }
+  }, []);
+
+  const toggleDialog = (ref) => {
+    if (!ref.current) {
+      return;
+    }
+    ref.current.hasAttribute("open")
+      ? ref.current.close()
+      : ref.current.showModal();
+  };
+
+  // Create reference to dom elements
+  const deleteDialogRef = useRef(null);
+  return (
+    <>
+      <Dialog isCustom={true} dialogRef={deleteDialogRef}>
+        <div className="flex flex-col items-center justify-center">
+          <p className="mb-6 text-center">
+            Get started by adding a friend to split expenses with.
+          </p>
+          <Button
+            className="w-full sm:w-48"
+            onClick={() => handleSetModal("FriendForm")}
```

Comment:
Can I recommend adding a navigate('/friends') in this onClick?

That way, when the user adds the new friend, it will show the friend list. Less whiplash as the user

Reply:
I thought it would be best to do it this way to get them right into the required task with less clicking.